zenfolio.com.customsubdomain.json

```diff
@@ -0,0 +1,18 @@
+{
+    "providerId": "Zenfolio",
+    "providerName": "Zenfolio Custom Subdomain-only Web Hosting",
+    "serviceId": "CustomSubDomain",
+    "serviceName": "Zenfolio Custom Subdomain-only Web Hosting",
+    "version": 2021111901,
+    "logoUrl": "https://zenfolio.com/resources/zenfolio-logo.gif",
+    "description": "This template is used for photographers to set up their custom subdomain to work with Zenfolio for web-hosting (only)",
+    "hostRequired": true,
+    "records": [
+        {
+            "type": "CNAME",
+            "host": "%host%",
```

Comment:
The template is automatically "rooted" according to the host variable, so here it would be correct to use `@`, otherwise some DNS providers would put the record at <host>.<host>.<domain>
Please refer to [the specification](https://github.com/Domain-Connect/spec/blob/master/Domain%20Connect%20Spec%20Draft.adoc#67-hostname-in-template) for details.

Reply:
I made the requested change, replacing %host% with @ in the template.  I also updated version numbers to reflect the current date on both files.